tooling/redhat-camel-spring-boot-bom/pom.xml

```diff
@@ -149,37 +149,37 @@
             <dependency>
                 <groupId>org.apache.tomcat</groupId>
                 <artifactId>tomcat-annotations-api</artifactId>
-                <version>10.1.33</version>
+                <version>10.1.34</version>
```

Comment:
Given that we are using the same version as spring boot, are these tomcat overrides needed?

Reply:
Probably not but I'd be in favor of keeping them in place because we've had to add them before due to CVE.

Reply:
okok, sounds good